File: api/code-samples/python/personal/sign.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"
message = "0xdeadbeaf"
password = "helloWorld"

sdk = OpenSDK(host)
personal_response = sdk.personal.sign(message, address, password)

print(personal_response)

File: web3rpc/rpc-specs/api/code-samples/javascript/personal/sign.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));


    const address = '0x6b1ac7bda0073095df1d434c39d66fb31a592bdd';
    const message = '0xdeadbeaf';
    const password = "helloWorld";

    sdk.personal.sign(message, address, password, {}, (err, data, response) => {
        console.log(data);
    });
}
)()